tools/cli/commands/installDependency.js

```diff
@@ -0,0 +1,17 @@
+const chalk = require('chalk');
+const { spawn } = require('child_process');
+
+function installDependency({ logger, moduleName }) {
```

Comment:
Why do we run `yarn` when we add module? It takes much time to run yarn. It was not needed before, why do we need it now? Why making symlink is not enough?

Reply:
Looks like it is not enough, since I get the error continuously ```spin error Error: Cannot find module '@module/example-client-react'``` after the module is added and `yarn watch` command run.

Reply:
Could you please test on your side as well?

Reply:
@evgenijolshanskij Ok, I will check and report back